sherlock/db/migrations/000058_chart_version_v3.down.sql

```diff
@@ -0,0 +1,4 @@
+DROP INDEX IF EXISTS chart_versions_selector_unique_constraint;
+
+ALTER TABLE v2_chart_versions
+    DROP CONSTRAINT IF EXISTS chart_version_present;
```

Comment:
```suggestion
    DROP CONSTRAINT IF EXISTS chart_version_present;

```
This is what's actually causing the test failure, there's a unit test that validates that these files have newlines at the end